### DISCUSSION TITLE
history_stats in formato sessagesimale

### DISCUSSION THREAD

**Discussion by dante70** [indomus/forum#2093] [GENERALE - Discussioni generiche]
salve a tutti,
ho un sensore history stats che  mi indica le ore di funzionamento caldaia.
Vorrei creare un sensore template che mi trasformasse il risultato in sessagesimale.
il tempo viene indicato in ore,ma ad esempio 1,1 ore sono1 ora +1/10 di ora e cioe' 76 minuti.
In questo esempio vorrei che mi risultasse 1 ore e  6 minuti o ancor meglio dire 76 minuti
La proporzione la so fare ma non so scrivere  il codice.
se qualcuno mi da una mano lo ringrazio in anticipo

**Comment by dante70** [comment]
scusate,stamattina ero assonnato,1 ora e 6 minuti sono 66 minuti

**Reply by 50494554524F** [reply]
```
{{ 
    ( states( 'quinomesensore' ) | float * 60 ) | int
        if states( 'quinomesensore' ) not in [ 'unknown', 'unavailable' ]
        else
    'unknown'
}}
```
oppure
```
{{ 
    iif( 
        ( states( 'quinomesensore' ) not in [ 'unknown', 'unavailable' ], 
        states( 'quinomesensore' ) | float * 60 ) | int, 
        'unknown'
    )
}}
```
in modo simile, se vuoi, puoi definire "availability" del nuovo sensore...

**Reply by dante70** [reply]
grazie
